.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please remove this file as I believe this is your own settings 
for launching the CATcher app :P